multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -90,6 +91,28 @@ def is_categorical_column(
         return False
 
 
+def is_rois_column(data: pd.Series) -> bool:
+    """
+    Identify if a column is one rois column.
+
+    Parameters
+    ----------
+    X
+        One column of a multimodal pd.DataFrame for training.
+
+    Returns
+    -------
+    Whether the column is a rois column.
+    """
+    idx = data.first_valid_index()
+    return (
```

Comment:
Will we support json encoded string?

E.g., `"{'xmin': ..., 'ymin': ..., 'xmax': ..., ...}"`

Reply:
User's input for evaluation will be a json file like "coco17/annotations/instances_val2017.json", and it's processed by utils.from_coco to get the dataframe. It's not necessary to add the support. There's also no harm but the code can not be tested in unit test at this point.